ci-operator/config/ComplianceAsCode/content/ComplianceAsCode-content-master__4.15.yaml

```diff
@@ -810,6 +810,106 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: e2e-aws-rosa-ocp4-pci-dss-node-weekly
+  cron: 0 22 * * 6
+  steps:
+    cluster_profile: quay-aws
+    env:
+      ADDITIONAL_SECURITY_GROUP: "true"
+      CHANNEL_GROUP: nightly
+      OCM_LOGIN_ENV: integration
+      OPENSHIFT_VERSION: "4.15"
+    test:
+    - as: test-ocp4-pci-dss-node
+      cli: latest
+      commands: |
+        export PROFILE=pci-dss-node
+        export PRODUCT=ocp4
+        export CONTENT_IMAGE=ghcr.io/complianceascode/k8scontent:latest
+        export component=ocp4-content-ds
+        git clone https://github.com/ComplianceAsCode/ocp4e2e.git ocp4e2e
+        pushd ocp4e2e
+        go test -v -timeout 120m github.com/ComplianceAsCode/ocp4e2e -profile="$PROFILE" -product="$PRODUCT" -content-image="$CONTENT_IMAGE"
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: rosa-aws-sts-hcp
```

Comment:
I'm struggling to understand how this is wired up to the trace we're getting from CI, which looks like:

```
INFO[2024-04-18T21:11:09Z] Logging into integration with offline token
ERR: Failed to create OCM connection: error creating connection. Not able to get authentication token: access and refresh tokens are unavailable or expired, and there are no password or client secret to request new ones 
```

The [rosa-aws-sts-hcp](https://steps.ci.openshift.org/workflow/rosa-aws-sts-hcp) has several steps but ultimately gets to a [rosa-cluster-provision](https://steps.ci.openshift.org/reference/rosa-cluster-provision) step. The error we should be getting from that script doesn't match what we're observing in CI:

https://steps.ci.openshift.org/reference/rosa-cluster-provision#line102

It looks like we're failing on the [rosa-sts-account-roles-create](https://steps.ci.openshift.org/reference/rosa-sts-account-roles-create#line30) step based on the log and the trace...

But I don't know how that's getting wired up when it's not in the `rosa-aws-sts-hcp` chain?

Reply:
I needed to implement SSO support in https://github.com/openshift/release/pull/51154 to get this further along with our CI service account.